pdns/recursordist/docs/lua-scripting/dq.rst

```diff
@@ -108,6 +108,12 @@ The DNSQuestion object contains at least the following fields:
 
       A string that will be used to set the ``deviceId`` field in :doc:`protobuf <../lua-config/protobuf>` messages.
 
+  .. attribute:: DNSQuestion.deviceName
```

Comment:
It looks like we don't actually export this field (or `deviceId` for that matter, look at `requestorId` if you want a working example) of the `DNSQuestion` object in the non-FFI Lua interface.

Reply:
Thanks for your review. Should be fixed now.